### PR TITLE
Adding cirrus test for FreeBSD

### PR DESCRIPTION
**Pull request by bimbashrestha**
Gmake test runs on init on my fork: 
https://github.com/bimbashrestha/lz4/pull/3

### CODE REVIEW

File: tests/test_install.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #/usr/bin/env sh
 set -e
 
-make="make -C $lz4_root"
+make="gmake -C $lz4_root"
```
Comment (Cyan4973) [inline tests/test_install.sh]: `gmake` is for FreeBSD, but Linux still uses `make`.

File: tests/test_install.sh [hunk 2]
```diff
@@ -1,7 +1,14 @@
 #/usr/bin/env sh
 set -e
 
+
 make="make -C $lz4_root"
+if [ "$unamestr" = 'Linux' ]; then
```
Comment (Cyan4973) [inline tests/test_install.sh:7]: where is defined `$unamestr` ?

Comment (bimbashrestha) [reply tests/test_install.sh:7]: Ahh good catch. Accidentally removed it before pushing.

**Review by Cyan4973 — APPROVED** [review]

**Comment by Cyan4973** [comment]
Thanks @bimbashrestha , everything seems to work fine on Cirrus CI !

**Pull request merged**